Content.Shared/Maps/ContentTileDefinition.cs

```diff
@@ -42,7 +43,7 @@ public sealed partial class ContentTileDefinition : IPrototype, IInheritingProto
         [DataField("isSubfloor")] public bool IsSubFloor { get; private set; }
 
         [DataField("baseTurf")]
-        public string BaseTurf { get; private set; } = string.Empty;
+        public List<string>? BaseTurf { get; private set; } = new();
```

Comment:
i think this should really be split into two datafields:
`BaseTurf`: A single prototype that is the tile that is underneath this one by default
`BaseWhitelist`: A list of tags used to determine if you can place this tile on another tile.

My issue with this implementation is that you are required to update all tiles every single time you make a variant. This is unacceptable maintenance-wise.

Also replace string with `ProtoId<ContentTileDefinition>`